src/const/const.ts

```diff
@@ -0,0 +1,46 @@
+export const RATING_NUMBER = 20;
+
+export const cities = [
+  {
+    city: 'Paris',
+  },
+  {
+    city: 'Cologne',
+  },
+  {
+    city: 'Brussels',
+  },
+  {
+    city: 'Amsterdam',
+  },
+  {
+    city: 'Hamburg',
+  },
+  {
+    city: 'Dusseldorf',
+  },
+];
+
+
+export const enum TypeCard {
+  Offer = 'OFFER',
+  Favorites = 'FAVORITES',
+  Place = 'PLACE',
+}
+
+export const enum AuthorizationStatus {
+  Auth = 'AUTH',
+  NoAuth = 'NOAUTH',
+  Unknown = 'UKNOWN',
+}
+
+export const offerInside: string[] = [ 'Wi-Fi', 'Washing machine', 'Towels', 'Heating', 'Coffee machine', 'Baby seat', 'Kitchen', 'Dishwasher', 'Cabel TV', 'Fridge' ];
+export const offerImage: string[] = ['img/room.jpg', 'img/apartment-01.jpg', 'img/apartment-02.jpg', 'img/apartment-03.jpg', 'img/studio-01.jpg', 'img/apartment-01.jpg'];
+
+export const URL_MARKER_DEFAULT = 'https://assets.htmlacademy.ru/content/intensive/javascript-1/demo/interactive-map/pin.svg';
+export const URL_MARKER_CURRENT = 'https://assets.htmlacademy.ru/content/intensive/javascript-1/demo/interactive-map/main-pin.svg';
+
+export const ICON_WIDTH = 40;
+export const ICON_HEIGHT = 40;
+export const ANCHOR_X = 20;
+export const ANCHOR_Y = 40;
```

Comment:
Это всё можно сгруппировать

```suggestion
export const IconSize = {
  WIDTH: 40,
  HEIGHT: 40,
}

export const AnchorCoords = {
  X: 20,
  Y: 40;
}
```